pkgs/games/onscripter-en/default.nix

```diff
@@ -4,12 +4,12 @@
 
 
 stdenv.mkDerivation {
-  name = "onscripter-en-20110930";
+  name = "onscripter-en-20171218";
 
   src = fetchurl {
     # The website is not available now.
-    url = "https://www.dropbox.com/s/ag21owy9poyr2oy/onscripter-en-20110930-src.tar.bz2";
-    sha256 = "1kzm6d894c0ihgkwhd03x3kaqqz0sb6kf0r86xrrz12y309zfam6";
+    url = "https://www.dropbox.com/s/ag21owy9poyr2oy/onscripter-en-20171218-src.tar.bz2";
```

Comment:
This links simply seems to redirect to the old one :thinking: .

Reply:
Yeah, it simply seems to be building the older version:

```
[nix-shell:~/.cache/nixpkgs-review/pr-110246]$ ./results/onscripter-en/bin/onscripter-en  --version
System info: Intel CPU, with functions: MMX SSE SSE2
ONScripter-EN version 20110930-en 'Yakisoba' (2.94)
Original written by Ogapee <ogapee@aqua.dti2.ne.jp>,
English fork maintained by "Uncle" Mion Sonozaki <UncleMion@gmail.com>

Copyright (c) 2001-2011 Ogapee, 2007-2011 Sonozaki
This is free software; see the source for copying conditions.
```

I think the bot confused the change in the `sha256` format with a new hash :thinking: ? But I still trying to understand how it detected this update.

Reply:
> This update was made based on information from https://repology.org/metapackage/onscripter-en/versions.

Reply:
TIL.

BTW, I was looking at the provided link, this is the "up-to-date" fork: https://github.com/superzazu/onscripter-en

Seems to be a macOS only fork, so it doesn't make sense to change the receipt to point to the new link either.